util/crc32c.cc

```diff
@@ -313,6 +313,9 @@ static inline void Slow_CRC32(uint64_t* l, uint8_t const **p) {
   table0_[c >> 24];
 }
 
+#if defined(HAVE_SSE42) && defined(__GNUC__)
+__attribute__ ((target ("sse4.2")))
```

Comment:
see http://releases.llvm.org/3.8.0/tools/clang/docs/AttributeReference.html#target-gnu-target and https://gcc.gnu.org/onlinedocs/gcc-4.8.0/gcc/Function-Attributes.html